### ISSUE TITLE
Combine strings involving variables causes problems.

### ISSUE THREAD

**Issue by ItsAquaLight** [microsoft/vscode#175985]
Type: <b>Bug</b>

(Create a new string) let testString = "This is a string";
(Log it) console.log(testString)
(The output) This is a string

(Create another string) let newString = `${testString}12345678`;

Pay attention to how the value for newString is formatted.

(Log the new string) console.log(newString);
(The output) 12345678a string

No idea when this bug developed but it was somewhere between the years 2020 and 2023.

VS Code version: Code 1.76.0 (92da9481c0904c6adfe372c12da3b7748d74bdcb, 2023-03-01T10:22:44.506Z)

**Comment by VSCodeTriageBot** [comment]
This issue has been closed automatically because it needs more information and has not had recent activity. See also our [issue reporting](https://aka.ms/vscodeissuereporting) guidelines.

Happy Coding!

**Comment by ItsAquaLight** [comment]
Apologies for letting this thread close. I found the issue in the end and it was the result of lack-of-experience. I'm primarily a C# coder and though Node.Js, which is what I was using to program a Discord bot, is similar to C#, the FileStream module I was using, to me, was more unfiltered. When a text file is read in C#, the only data for new lines were a new line character while in node it had a carriage return before each new line character. I swore I checked for it but with little success. Never knew it was so hard to find a carriage return, lol.

**Comment by ItsAquaLight** [comment]
> This sounds like an issue with your code. If you think it's a vscode issue then please include full test code and show how it reproes in vscode but not when running your code directly from a terminal.

Apologies for letting this thread close. I found the issue in the end and it was the result of lack-of-experience. I'm primarily a C# coder and though Node.Js, which is what I was using to program a Discord bot, is similar to C#, the FileStream module I was using, to me, was more unfiltered. When a text file is read in C#, the only data for new lines were a new line character while in node it had a carriage return before each new line character. I swore I checked for it but with little success. Never knew it was so hard to find a carriage return, lol.
